fabio.properties

```diff
@@ -813,3 +813,19 @@
 # The default is
 #
 # ui.title =
+
+
+# ui.certfile configures the UI/API with a TLS server certificate.
+# If this is specified, the UI/API will be served over HTTPS.
+#
+# The default  is
```

Comment:
I see two spaces `default__is`. Also, default should be `ui.certfile = `. Same below.

```
# ui.certfile contains the path to the TLS certificate for the UI/API endpoint
# which enables serving the UI and API over HTTPS.
#
# If the file containing the certificate also contains the private key then the
# ui.keyfile option can be omitted.
#
# The default is
#
# ui.certfile =
```